aten/src/ATen/NestedTensorImpl.h

```diff
@@ -169,6 +169,8 @@ struct TORCH_API NestedTensorImpl : public c10::TensorImpl {
   //       we can compute it from `nested_sizes_`
   mutable c10::optional<std::vector<int64_t>> opt_sizes_;
 
+  mutable c10::optional<std::vector<c10::SymInt>> sym_sizes_;
```

Comment:
This is a pre-existing condition but what's going on with these being mutable fields?